images/linux/scripts/installers/action-archive-cache.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
+################################################################################
+##  File:  action-archive-cache.sh
+##  Desc:  Download latest release from https://github.com/actions/action_verions
+##         and un-tar to /opt/actionarchivecache
+################################################################################
+
+# Prepare directory and env variable for ACTIONS_RUNNER_ACTION_ARCHIVE_CACHE
+ACTION_ARCHIVE_CACHE_DIR=/opt/actionarchivecache
+mkdir -p $ACTION_ARCHIVE_CACHE_DIR
+echo "ACTIONS_RUNNER_ACTION_ARCHIVE_CACHE=$ACTION_ARCHIVE_CACHE_DIR" | tee -a /etc/environment
+chmod -R 777 $ACTION_ARCHIVE_CACHE_DIR
+
+# Download latest release from github.com/actions/action-versions and untar to /opt/actionarchivecache
+pushd $ACTION_ARCHIVE_CACHE_DIR
+latestReleaseName=$(curl -s -H "Accept: application/vnd.github+json" -H "X-GitHub-Api-Version: 2022-11-28" https://api.github.com/repos/actions/action-versions/releases/latest | jq --raw-output  '.name')
```

Comment:
I'd use the `get_github_package_download_url` helper which we have differed in the runner-images repo here: https://github.com/actions/runner-images/blob/a4234d46c7b5db3e2187af0db44790e1a898c802/images/linux/scripts/helpers/install.sh#L69C1-L69C33